src/platform-implementation-js/style/gmail.css

```diff
@@ -842,13 +829,13 @@ input.inboxsdk__x_close_button {
 }
 
 .inboxsdk__gmail_action:active {
-  box-shadow: inset 0 1px 2px rgba(0, 0, 0, 0.1);
+  box-shadow: inset 0 1px 2px rgb(0 0 0 / 10%);
 }
 
 .inboxsdk__gmail_action:hover {
-  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.05);
+  box-shadow: 0 1px 1px rgb(0 0 0 / 5%);
   background-color: #ededed;
-  background-image: -webkit-linear-gradient(top, #ededed, #eaeaea);
```

Comment:
From chatting with @dmy7r0 offline (https://caniuse.com/?search=linear-gradient):

> Partial support in Safari and Older Firefox versions refers to not using premultiplied colors which results in unexpected behavior when using the transparent keyword as advised by the [spec](https://www.w3.org/TR/2012/CR-css3-images-20120417/#color-stop-syntax).

Because we support Safari 15 for now, we can just let the existing change be with a stylelint disable to get this through.

Reply:
Reverted in f48defd4fab71f8ad38da3f6b0166d56c615994b